calendar-cli.py

```diff
@@ -11,8 +11,8 @@
 from datetime import datetime, timedelta, date
 import dateutil.parser
 from dateutil.rrule import rrulestr
-from icalendar import Calendar,Event,Todo,Journal
-import caldav
+from icalendar import Calendar,Event,Todo,Journal,Alarm
+#import caldav
```

Comment:
No trouble.  Thanks for catching my silly mistake; I think that was the result of a poorly-handled merge conflict or something.  I've put the import back in ('cause you were right -- it broke everything) and I'll try to be less reckless in the future.

Thanks for the great tool, by the way; I love using Lightning, but I'm about to make about 60 TODOs for the course I'm teaching and it'll be much more pleasant this way!